tests/test_sd.py

```diff
@@ -3,7 +3,7 @@
     BJMMpdw, BothMay, MayOzerov, MayOzerovD2, MayOzerovD3, Stern, BJMM_plus
 from cryptographic_estimators.SDEstimator import SDProblem
 from math import log2
-from module.estimator import prange_complexity, dumer_complexity, stern_complexity, ball_collision_decoding_complexity, \
+from .module.estimator import prange_complexity, dumer_complexity, stern_complexity, ball_collision_decoding_complexity, \
     bjmm_depth_2_complexity, bjmm_depth_3_complexity, bjmm_complexity, bjmm_depth_2_disjoint_weight_complexity, \
     bjmm_depth_2_partially_disjoint_weight_complexity, both_may_depth_2_complexity, may_ozerov_complexity, \
     may_ozerov_depth_2_complexity, may_ozerov_depth_3_complexity
```

Comment:
https://github.com/Crypto-TII/cryptographic_estimators/blob/403ee0cc4f933e7de72ea7a48aa63cfd762fa351/tests/test_sd.py#L10

This import can not be resolve because optimize does not exits in module @FloydZ